### DISCUSSION TITLE
video plugin for inkscape

### DISCUSSION THREAD

**Discussion by Rocco83** [senshu/Sozi#658] [Q&A]
Hi,

I am following the documentation:
https://sozi.baierouge.fr/pages/tutorial-media.html

I do have Inkscape 1.2 (dc2aeda, 2022-05-15) on OS X.

I've copied out the plugin (v1.0) into the directory as requested.
~~~
daniele$ ls -la .config/inkscape/extensions/
total 40
drwxr-x--x   5 daniele  staff   160 20 Ott 07:57 .
drwxr-x--x  10 daniele  staff   320 20 Ott 07:57 ..
-rw-r--r--@  1 daniele  staff  6148 20 Ott 08:05 .DS_Store
-rw-rw-r--@  1 daniele  staff  1903  2 Mag  2021 sozi_extras_media.inx
-rw-rw-r--@  1 daniele  staff  4120  2 Mag  2021 sozi_extras_media.py
daniele$ 
~~~

And restarted Inkscape.

Unfortunately, the Sozi menu does not show up.

Is think a known issue because of the Inkscape version, or...

Thank you very much,
Daniele

**Comment by aumouvantsillage** [comment]
I have not tested this extension with Inkscape 1.2.
I'll try and see if I have the same problem.

**Reply by Gideon5L2F** [reply]
I have just found this thread. I would like to have an audio file accompanying my Sozi presentation but extracting the extention produces three errors as follows:

Cannot create folder : C:\Program Files\Inkscape\share\extensions\media-inkscape-1.0
Cannot open output file : The system cannot find the path specified. : C:\Program Files\Inkscape\share\extensions\media-inkscape-1.0\sozi_extras_media.inx
Cannot open output file : The system cannot find the path specified. : C:\Program Files\Inkscape\share\extensions\media-inkscape-1.0\sozi_extras_media.py

My Inkscape version is 1.1.  Has this glitch been fixed yet?

Thanks

**Reply by aumouvantsillage** [reply]
@Gideon5L2F As far as I can tell, this is not an issue that I can fix.
Something is preventing you from extracting the files to the desired location.
Maybe I could help if you described the actions that led to these errors.

**Comment by kingigi** [comment]
I believe that it's no longer possible to use Sozi as a plugin on Inkscape version 1.2.
You need to create an SVG using Inkscape, save it, and use the separate sozi executable to generate your animated presentation outside of Inkscape.

**Comment by aumouvantsillage** [comment]
In my last attempt, I could install and use the extension successfully in Inkscape 1.2 on Linux.
As expected, I got a *Sozi extras* submenu in the Inkscape *Extensions* menu.

It could be an issue specific to OS X.
